app/services/triannon/ldp_loader.rb

```diff
@@ -99,7 +99,7 @@ def get_ttl sub_path = nil
     def statements_from_ttl_minus_fedora ttl
       # RDF::Turtle::Reader.new(ttl).statements.to_a
       g = RDF::Graph.new.from_ttl(ttl) if ttl
-      RDF::FCRepo4.remove_fedora_triples(g).statements if g
+      OA::Graph.remove_fedora_triples(g).statements if g
```

Comment:
when Triannon serves out an anno, it shouldn't have any Fedora or LDP triples in it, as they are an artifact of the storage of the anno, not the anno itself.  SearchWorks, Mirador, etc. shouldn't receive annos with backend noise in them.  That's why we did this.